app/main/posts/views/post-filters.service.js

```diff
@@ -100,14 +102,32 @@ function PostFiltersService(_, FormEndpoint, TagEndpoint, $q) {
     }
 
     function getQueryParams(filters) {
+        var defaults = getDefaults();
         var query = _.omit(
             filters,
             function (value, key, object) {
+                if (key === 'saved_search') {
+                    return true;
+                }
+                if (key === 'reactiveFilters') {
+                    return true;
+                }
+                if (key === 'qEnabled') {
+                    return true;
+                }
                 // Is value empty?
                 // Is it a date?
                 if (_.isDate(value)) {
                     return false;
                 }
+
+                // Is an array with all the same elements? (order doesn't matter)
```

Comment:
we don't want to send category or surveys when we have the same as the default values because that might cause backend issues specially around things that dont have categories